pkg/api/v1/schema.go

```diff
@@ -758,3 +762,18 @@ type PodNetwork struct {
 	// Default 10.0.2.0/24 if not specified.
 	VMNetworkCIDR string `json:"vmNetworkCIDR,omitempty"`
 }
+
+// Rng represents the random device passed from host
+// ---
+// +k8s:openapi-gen=true
+type Rng struct {
+	// Source sets the host source used to provide entropy
+	Source string `json:"source"`
+	// RateBytes limit the amount of bytes that the VM can read from entropy source
+	// per period
+	// +optional
+	RateBytes *uint32 `json:"rateBytes,omitempty"`
+	// RatePeriod sets the length of a read period
+	// +optional
+	RatePeriod *uint32 `json:"ratePeriod,omitempty"`
+}
```

Comment:
Are you planning to remove the RateBytes and RatePeriod in the next iteration? 
I thought we've agreed not to expose RateBytes and RatePeriod to the users as it introduces a security issue.